core/src/main/scala/cromwell/core/DockerConfiguration.scala

```diff
@@ -1,35 +1,26 @@
 package cromwell.core
 
-import com.typesafe.config.ConfigFactory
-
-import scala.concurrent.duration.FiniteDuration
 import cats.data.Validated._
 import cats.syntax.apply._
+import com.typesafe.config.{Config, ConfigFactory}
 import common.exception.AggregatedMessageException
 import common.validation.ErrorOr.ErrorOr
-import net.ceedubs.ficus.Ficus._
 import common.validation.Validation._
-import cromwell.core.io.Throttle
+import net.ceedubs.ficus.Ficus._
 import org.slf4j.LoggerFactory
 
-import scala.concurrent.duration._
+import scala.concurrent.duration.FiniteDuration
 
 object DockerConfiguration {
   private val logger = LoggerFactory.getLogger("DockerConfiguration")
-  lazy val dockerConfig = ConfigFactory.load().getConfig("docker")
-  lazy val dockerHashLookupConfig = dockerConfig.getConfig("hash-lookup")
+  lazy val dockerConfig: Config = ConfigFactory.load().getConfig("docker")
+  lazy val dockerHashLookupConfig: Config = dockerConfig.getConfig("hash-lookup")
 
   lazy val instance: DockerConfiguration = {
     if (dockerHashLookupConfig.hasPath("gcr-api-queries-per-100-seconds")) {
-      logger.warn("'docker.hash-lookup.gcr-api-queries-per-100-seconds' is being deprecated, use 'docker.hash-lookup.gcr.throttle' instead (see reference.conf)")
+      logger.warn("'docker.hash-lookup.gcr-api-queries-per-100-seconds' is no longer supported, use 'docker.hash-lookup.google.throttle' instead (see reference.conf)")
```

Comment:
ToL: Confirming that this is just a warning? Suppose someone thought they had turned this on in the past. They don't read the changelog nor the log messages and upgrade to 66. Nothing majorly bad will happen, right?

Reply:
Nothing bad will happen if this is set. Cromwell used to ignore this parameter by mistake, now it ignores it on purpose.